proto/storage/process_listening_on_port.proto

```diff
@@ -50,4 +50,7 @@ message ProcessListeningOnPortStorage {
 
     string process_indicator_id = 5
         [(gogoproto.moretags) = 'search:"Process ID,store" sql:"fk(ProcessIndicator:id),no-fk-constraint,index=btree,type(uuid)"'];
+
+    // XXX: Make it a partial index on only active, not closed, PLOP
+    bool closed = 6 [(gogoproto.moretags) = 'search:"Closed,store"  sql:"index=btree"'];
```

Comment:
I don't think you need this additional field. The close_timestamp already keeps track of if the port is open or closed. If close_timestamp is nil the port is open. If close_timestamp is not nil the port is closed.

Reply:
`close_timestamp` is part of serialized field and is not extracted into a separate column to operate via sql. I was thinking about this option, but decided to use a boolean value instead because we have to create an index on it, and in this way it would be smaller than an index over a timestamp field (especially when partial index support will be introduced into the generator).